examples/metrics/prometheus-install/prometheus-rules.yaml

```diff
@@ -185,6 +185,46 @@ spec:
       annotations:
         summary: 'All Kafka Bridge containers down or in CrashLookBackOff status'
         description: 'All Kafka Bridge containers have been down or in CrashLookBackOff status for 3 minutes'
+    - alert: AvgProducerLatency
+      expr: strimzi_bridge_kafka_producer_request_latency_avg > 10
+      for: 10s
+      labels:
+        severity: warning
+      annotations:
+        summary: 'Kafka Bridge average consumer fetch latency'
+        description: 'The average fetch latency is {{ $value }} on {{ $labels.clientId }}'
+    - alert: AvgConsumerFetchLatency
+      expr: strimzi_bridge_kafka_consumer_fetch_latency_avg > 10
+      for: 10s
+      labels:
+        severity: warning
+      annotations:
+        summary: 'Kafka Bridge consumer average fetch latency'
+        description: 'The average consumer commit latency is {{ $value }} on {{ $labels.clientId }}'
+    - alert: AvgConsumerCommitLatency
+      expr: strimzi_bridge_kafka_consumer_commit_latency_avg > 10
+      for: 10s
+      labels:
+        severity: warning
+      annotations:
+        summary: 'Kafka Bridge consumer average commit latency'
+        description: 'The average consumer commit latency is {{ $value }} on {{ $labels.clientId }}'
```

Comment:
Out of curiosity, how did you come up with the value of 10 (ms I guess?)?

Reply:
As far as I can see they are "seconds" and 10 seconds of latency sound to be too high for an alarm.

Reply:
It should be miliseconds. So if we have latency > 10ms for 10s, fire an alert.

Reply:
So have you done any tests to see that the 10ms latency is really something to alert about?

Reply:
No, I do not know how to simulate latency

Reply:
I guess you should at lest try some performance test and watch what will be the average latency during it to set the alert right.